lib/ansible/modules/cloud/vmware/vmware_guest_network.py

```diff
@@ -332,7 +332,7 @@ def sanitize_network_params(self):
                     if network['state'].lower() == 'new' and not network['start_connected']:
                         network['connected'] = False
                 # specified network does not exist
-                if 'name' in network and not self.network_exists_by_name(network['name']):
+                if 'name' in network and not find_network_by_name(self.content, 'VM Network'):
```

Comment:
Why we are hard-coding network name to `VM Network` ?

Reply:
:man_facepalming: